Resources/string.csv

```diff
@@ -79,6 +79,7 @@ GM,GM,GM(Призрак),GM,GM(管理员),GM(遊戲大師)
 
 # 属性
 LastImpostor,Last Impostor,Последний Предатель,ラストインポスター,绝境者,絕境者
+LastImpostorKillCooldown,LastImpostorKillCooldown,,,绝境者击杀冷却时间
```

Comment:
not require

Reply:
If it's removed, the host setting would show an error message like<INVALID: LastImpostorKillCooldown>

Reply:
![image](https://user-images.githubusercontent.com/76250388/190845144-f0f46f04-30aa-42be-9950-d2e93d218c63.png)
``LastImpostorKillCooldown``  isn't use now.

Reply:
sorry. I'll fix it.
Just a minute.
After fix merged, do ``fetch`` and ``pull``.

Reply:
Done